packages/reports/src/Constants/routes-config.tsx

```diff
@@ -10,14 +11,16 @@ const lazyLoadReportComponent = makeLazyLoader(
     () => <Loading />
 );
 
-type TRoute = {
+export type TRoute = {
```

Comment:
Can this be placed in a separate file like `routes.types.ts` I believed that we are using `TRoute`  elsewhere aswell.

Reply:
> Can this be placed in a separate file like `routes.types.ts` I believed that we are using `TRoute` elsewhere aswell.

I thought about this, but I've decided it will be done during refactor. If you think I can do this with this PR, I'll do it, no problem